assets/css/admin/style.scss

```diff
@@ -669,24 +669,30 @@ table.edd-order-overview-summary--refund {
 }
 
 /** Totals */
-.edd-order-overview-summary__totals .column-primary {
+.edd-order-overview-summary__subtotal .column-primary,
+.edd-order-overview-summary__tax tr:first-of-type .column-primary,
+.edd-order-overview-summary__total .column-primary {
 	font-weight: 600;
 }
 
-.edd-order-overview-summary__totals-total td {
+.edd-order-overview-summary__subtotal td,
```

Comment:
I know the vertical alignment is not new to this PR but I'm thinking I actually prefer it aligned top, except for the total row (because of the difference in font sizes).